src/nowsecure-action.ts

```diff
@@ -1,139 +0,0 @@
-/*
```

Comment:
Why is this being copied/moved/deleted out of `src`?

Reply:
Yes.  We now hae nowsecure-upload-action.ts, and nowsecure-sarif-action.ts.  nowsecure-action.ts was the one action that we had previously, and I broke it into two actions so that the environment could trigger the second one after the wait delay.